libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSDKCryptoUtils.m

```diff
@@ -424,4 +444,70 @@ +(nullable SecKeyRef)getRSAKeyRefWithTag:(NSString *)keyTagString keyLength:(NSU
     return keyRef;
 }
 
++ (NSData *)getRSAPublicKeyAsDER:(NSData *)keyData {
```

Comment:
Did we originate this code?  If this is based on someone else's work, we need to attribute it, make sure we have a license for it, etc.

Reply:
I added the original blog link at comment header here but I may miss other places.
https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/2392/files#diff-af9b3c572bb034509566bf659be0424fR95

Is there anywhere we also need to add additional comments?

Reply:
No, I just missed it there.  Thanks for pointing it out, @liguopku !